Code/Mantid/Vates/VatesSimpleGui/ViewWidgets/src/SplatterPlotView.cpp

```diff
@@ -466,8 +468,7 @@ void SplatterPlotView::onRemovePeaksTable()
 
   if (m_peaksFilter)
   {
-    pqObjectBuilder *builder = pqApplicationCore::instance()->getObjectBuilder();
-    builder->destroy(m_peaksFilter);
+    this->destroyFilter(QString("MDPeaksFilter"));
```

Comment:
Lets use static const's for these repeated identifier names, also, shouldn't need to explicitly construct a QString like this.